README.md

```diff
@@ -30,7 +30,6 @@ Video.js Compatibility: 7.x, 8.x
 - [Compatibility](#compatibility)
   - [Browsers which support MSE](#browsers-which-support-mse)
   - [Native only](#native-only)
-  - [Flash Support](#flash-support)
```

Comment:
This removal happened due to adding new API docs for some reason. 🤷‍♂️

Reply:
we probably forgot to run `npm run docs` (or `npm run docs:toc`) previously.